### ISSUE TITLE
Disabled apps have no contextRoot param after restart

### ISSUE THREAD

**Issue by ratcashdev** [payara/Payara#747]
Steps to reproduce:
1. asadmin deploy --enabled=false myapp.war
2. call REST: http://localhost:4848/management/domain/applications/application/sampleApp/get-context-root.xml?modulename=sampleApp The response contains the `<entry key="contextRoot" value="/MyApp"/>` entry
3. restart glassfish
4. call REST: http://localhost:4848/management/domain/applications/application/sampleApp/get-context-root.xml?modulename=sampleApp At this point the response does not contain the `<entry key="contextRoot" value="/MyApp"/>`. Moreover, this results in a masked NPE in 
```
Severe:   java.lang.NullPointerException
	at org.glassfish.admingui.common.handlers.ApplicationHandlers.getSubComponents(ApplicationHandlers.java:182)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```
and the modules of the app are not shown in the admin-gui

**Comment by OndrejM** [comment]
I was able to reproduce with a simple WAR app ([myapp.war.zip](https://github.com/payara/Payara/files/325165/myapp.war.zip))

However, I'm not sure whether this is a bug or just an enhancement request. It seems that disabled applications don't have any properties accessible and the properties are loaded only once an application is deployed. The reason why the contextRoot is available until application server restart is because it is a side-effect of previous deployment (event when the application was deployed as disabled, its configuration is available until server restart).

There is a bug in admin console when displaying modules of disabled apps though. Admin console does not expect that properties are not loaded, and raises the above exception. A simple nunll check would probably fix that.

Whether application properties (including context root) can be loaded even for disabled applications is a question.

**Comment by smillidge** [comment]
Fixed by #934
